man/man8/zpool.8

```diff
@@ -717,6 +717,25 @@ The default value is
 .Sy off .
 This property can also be referred to by its shortened name,
 .Sy listsnaps .
+.It Sy multihost Ns = Ns Sy on Ns | Ns Sy off
+Controls whether a pool activity check should be performed during
+.Nm zpool Cm import .
```

Comment:
maybe we could say, `...should be performed when the pool is opened (e.g. by "zpool import")`